functions/go/template/README.md

```diff
@@ -0,0 +1,39 @@
+# some-function-name
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Explain what this function does in one or two sentences.
+
+[//]: <> (Note: The content between `<!--mdtogo:Short-->` and the following
+`<!--mdtogo-->` will be used as the short description for the command.)
+
+<!--mdtogo-->
+
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+Explain the function config and behavior for this function in detail. Explain
+how to use the function in both declarative and imperative flows. If the
+function supports both ConfigMap and CRD as function config, both should be
+documented.
+
+[//]: <> (Note: The content between `<!--mdtogo:Long-->` and the following
+`<!--mdtogo-->` will be used as the long description for the command.)
+
+<!--mdtogo-->
+
+### Examples
+
+<!--mdtogo:Examples-->
+
+Add a link to the web page of the examples. Examples should be added in the
+`examples` directory.
```

Comment:
What `examples` directory? Does this README apply to any function or only functions in the catalog where we have this directory convention?

Reply:
Updated.